test/common/secret/secret_manager_impl_test.cc

```diff
@@ -1011,6 +1017,68 @@ name: "signing_key"
   checkConfigDump(TestEnvironment::substitute(expected_config_dump));
 }
 
+TEST_F(SecretManagerImplTest, SdsDynamicSecretPrivateKeyProviderUpdateSuccess) {
```

Comment:
Please add `//` description of what this test does above it.

Reply:
Fixed.